projects/ngx-paypal-lib/src/lib/models/paypal-models.ts

```diff
@@ -293,7 +299,7 @@ export type ShippingPreference =
 
 export type PaypalLandingPage = "LOGIN" | "BILLING";
 
-export type OrderIntent = "CAPTURE" | "AUTHORIZE" | "subscription";
+export type OrderIntent = "CAPTURE" | "AUTHORIZE" | "subscription" | "tokenize";
```

Comment:
We use `intent: "tokenize"` with `createBillingAgreement` - [here's paypal's docs](https://developer.paypal.com/sdk/js/configuration/#link-intent)

I didn't add any validation in this PR - the PayPal SDK will complain if we don't pass `intent=tokenize` and `vault=true` with `createBillingAgreement`, though